src/index.js

```diff
@@ -26,7 +25,8 @@ class ProjectionGridReact extends React.Component {
 }
 
 ProjectionGridReact.propTypes = {
-  records: PropTypes.arrayOf(PropTypes.any).isRequired,
+  data: PropTypes.arrayOf(PropTypes.any).isRequired,
```

Comment:
This is not required

Reply:
I feel it's better to have a `config` and a `projections` config at the top level to avoid name conflict. Projections may add more stuff to top level config. We cannot list them all here.

Reply:
I dont think so. Flat is better than nested. And we should let user feel more comfortable to understand the props compared to the react components they see everyday.